### DISCUSSION TITLE
Welcome to mix.core Discussions!

### DISCUSSION THREAD

**Discussion by Smilefounder** [mixcore/mix.core#283] [Ideas & Features]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by NagarRahul** [comment]
Hi
I am trying to use editor for the design page and after the design, content is not rendered on the page preview.

**Reply by Smilefounder** [reply]
Hi @NagarRahul ,

You can add the following code to show the page content:
Content: @Model.Content

![image](https://user-images.githubusercontent.com/3785721/106723466-b7ef1480-6639-11eb-82ab-cb1f8b939c30.png)

Thanks.

**Reply by NagarRahul** [reply]
**Hi, I have applied for this and it not rendered as formatted look like**
![image](https://user-images.githubusercontent.com/42827466/106730993-fdf5a980-6634-11eb-9099-49bdccd3853c.png)


**After saving a preview like this.**
![image](https://user-images.githubusercontent.com/42827466/106731212-3dbc9100-6635-11eb-9348-81d05d86b927.png)

It shows HTML tag

**Reply by Smilefounder** [reply]
Hi @NagarRahul I have split your question to Q&A section here https://github.com/mixcore/mix.core/discussions/326

**Comment by NagarRahul** [comment]
How we can integrate another WIGYWIG editor like GrapeJs (https://grapesjs.com/) with this CMS?

**Reply by Smilefounder** [reply]
Hi @NagarRahul I have split your question to Q&A section here: https://github.com/mixcore/mix.core/discussions/327

Kindly create a new topic in Q&A section for each question.

Thanks. :)
